ccx_messaging/publishers/rule_processing_publisher.py

```diff
@@ -107,13 +106,14 @@ def publish(self, input_msg, report):
 
         try:
             msg_timestamp = input_msg["timestamp"]
+            msg_version = report.pop("version", 0)
```

Comment:
Is the `"version"` something new provided by Insights engine?? IIRC this "schema version" is relative to our message format (OrgID, AccountNumber, etc), not the format of the "Report" value

Reply:
Coming back to this, the `version` is a new field in the insights engine's response when [using the new formatter](https://gitlab.cee.redhat.com/ccx/ccx-data-pipeline/-/merge_requests/743). But it is a field that neither we (our pipeline) nor our API's customers are interested in,

As I mentioned in this PR's description, by removing this field from the report, we keep the existing contracts (we keep providing the same response to existing smart-proxy v1 endpoints) without messing with the internals of our pipeline.

If we want to use that version as a replacement of the `outdata_schema_version` or not is open for discussion, but I think it makes sense.

Reply:
Ok, it makes sense to me now.

Feel free to create a task to start ignoring the "Version" field in the consumers of this messages